From fc56e665c2dc6aff0006269e38553dcb1d08b2d5 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 16 Dec 2011 13:20:40 +0800
Subject: [PATCH 182/256] ENGR00170455 IPU lib:Change head file for flash

This patch changes IPU lib head file to make
existing flash binary image work.
This is code change only which doesn't modify
code logic.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 ipu/mxc_ipu_hl_lib.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/ipu/mxc_ipu_hl_lib.h b/ipu/mxc_ipu_hl_lib.h
index 4aed0db..fbb989f 100644
--- a/ipu/mxc_ipu_hl_lib.h
+++ b/ipu/mxc_ipu_hl_lib.h
@@ -318,14 +318,15 @@ typedef struct {
         void * ovbuf_start[2];
         void * ovbuf_alpha_start[2];
 	void * outbuf_start[3];
-	void * rotbuf_phy_start[2];
 	int ifr_size;
 	int ovfr_size;
 	int ovfr_alpha_size;
 	int ofr_size;
-	int rotfr_size;
 
 	void * priv;
+
+	void * rotbuf_phy_start[2];
+	int rotfr_size;
 } ipu_lib_handle_t;
 
 /*!
-- 
1.8.0

